aml/org.testeditor.aml.dsl.tests/src/org/testeditor/aml/dsl/tests/parser/validation/ValidationTest.xtend

```diff
@@ -53,13 +51,115 @@ class ValidationTest extends AbstractValidationTest {
 		input.parse(ValueSpace).assertError(
 			REG_EX_VALUE_SPACE,
 			REG_EX_VALUE_SPACE__EXPRESSION__INVALID,
-			'''
-				The given expression is not a valid regular expression in Java:
-				java.util.regex.PatternSyntaxException: Illegal repetition near index 0
-				${
-				^
-			'''.toString.trim
+			"The given expression is not a valid regular expression in Java:",
+			"java.util.regex.PatternSyntaxException: Illegal repetition near index 0",
+			"${",
+			"^"
 		)
 	}
 
+	@Test
+	def void testUsageOfDefaultLocatorStrategy() {
+		// given
+		val input = '''
+			package org.test
+
+			import org.testeditor.dsl.common.testing.DummyLocatorStrategy
+			import org.testeditor.dsl.common.testing.DummyFixture
+
+			interaction type click {
+				label = "Click on"
+				template = "click on" ${element}
+				method = DummyFixture.clickOn(element, locatorStrategy)
+				locatorStrategy = SINGLE
+			}
+		'''
+
+		// when
+		val amlModel = input.parse
+		amlModel.assertNoErrors
+		val interactionType = amlModel.interactionTypes.head
+
+		// then
+		interactionType.locatorStrategy.qualifiedName.assertEquals(
+			"org.testeditor.dsl.common.testing.DummyLocatorStrategy.SINGLE")
+	}
+
+	@Test
+	def void testUsageOfElementOverDefaultLocatorStrategy() {
+		// given
+		val input = '''
+			package org.test
+
+			import org.testeditor.dsl.common.testing.DummyLocatorStrategy
+			import org.testeditor.dsl.common.testing.DummyFixture
+
+			interaction type click {
+				label = "Click on"
+				template = "click on" ${element}
+				method = DummyFixture.clickOn(element, locatorStrategy)
+				locatorStrategy = SINGLE
+			}
+
+			element type Button {
+				interactions = click
+			}
+
+			component type Dialog { }
+
+			component NewDialog is Dialog {
+				element NewButton is Button {
+					locator = "ok"
+					locatorStrategy = ID
+				}
+			}
+		'''
+
+		// when
+		val amlModel = input.parse
+		amlModel.assertNoErrors
+		val elementNewButton = amlModel.components.head.elements.head
+
+		// then
+		elementNewButton.locatorStrategy.qualifiedName.assertEquals(
+			"org.testeditor.dsl.common.testing.DummyLocatorStrategy.ID")
+	}
+
+	@Test
+	def void testMissingLocatorStrategy() {
```

Comment:
There should be one more test here that checks if the fixtures does not require a locator strategy. We don't want an error in this case.